akka-bbb-apps/build.sbt

```diff
@@ -56,7 +54,10 @@ scalaVersion := "2.13.9"
 // https://github.com/muuki88/sbt-native-packager-examples/tree/master/akka-server-app
 // http://www.scala-sbt.org/sbt-native-packager/index.html
 //-----------
-mainClass := Some("org.bigbluebutton.Boot")
```

Comment:
The `mainClass` setting was removed because it was not being explicitly used by any packaging or execution commands in the project. The application continues to function correctly without it, as the necessary entry point is likely determined automatically by the plugins in use.